gnucash/gtkbuilder/gnc-plugin-page-budget.glade

```diff
@@ -805,4 +817,119 @@
       <action-widget response="-7">close_button</action-widget>
     </action-widgets>
   </object>
+  <object class="GtkDialog" id="budget_note_dialog">
+    <property name="can_focus">False</property>
+    <property name="border_width">5</property>
+    <property name="title" translatable="yes">Budget notes</property>
```

Comment:
Small nitpick: I believe in English titles each word is typically capitalized, so this should become `Budget Notes`.